frontend-project/src/pages/Welcome.tsx

```diff
@@ -1,21 +1,21 @@
-import React from 'react';
+import React, { FC, ReactNode } from 'react';
 import { PageHeaderWrapper } from '@ant-design/pro-layout';
 import { Card, Typography, Alert } from 'antd';
 import styles from './Welcome.less';
 
-const CodePreview = ({ children }) => (
+const CodePreview: FC<{}> = ({ children }) => (
   <pre className={styles.pre}>
     <code>
       <Typography.Text copyable>{children}</Typography.Text>
     </code>
   </pre>
 );
 
-export default () => (
+export default (): ReactNode => (
   <PageHeaderWrapper>
     <Card>
       <Alert
-        message="umi ui 现已发布，点击右下角 umi 图标即可使用"
+        message="User interfaces is now aviable. Click on icon on bottom-right to use it." // creditsy dla google translate
```

Comment:
tutaj albo `interface` ale `are`, ale to pewnie poprawimy, jak będziemy przenosić treści do `locales`.

Reply:
Sorry, walnąłem się. Poprawię przy okazji któregoś innego pr